script/build

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env node
+
+'use strict';
+
+let rollup = require('rollup').rollup;
+let babel = require('rollup-plugin-babel');
+let babelrc = require('babelrc-rollup').default;
+let filesize = require('file-size');
+let gzipSize = require('gzip-size').sync;
+let stat = require('fs').statSync;
+let minify = require('uglify-js').minify;
+let pkg = require('../package.json');
+
+rollup({
```

Comment:
This is likely to be the most controversial part of this PR and would be mostly obviated by https://github.com/rollup/rollup/issues/655. The biggest reason for this is that it speeds up the build since you're not doing the work twice, but it also removes three config files in the root of the project.

@Rich-Harris What do you think?